src/OSPSuite.Core/Domain/Formulas/DistributionFormulaFactory.cs

```diff
@@ -21,68 +25,85 @@ public interface IDistributionFormulaFactory
          IParameter minParameter,
          IParameter maxParameter);
 
+      DistributionFormula CreateFor(DistributionType distributionType, IDimension dimension);
    }
 
    public class DistributionFormulaFactory : IDistributionFormulaFactory
    {
       private readonly IObjectPathFactory _objectPathFactory;
       private readonly IObjectBaseFactory _objectBaseFactory;
 
-      public DistributionFormulaFactory(IObjectPathFactory objectPathFactory,IObjectBaseFactory objectBaseFactory)
+      public DistributionFormulaFactory(IObjectPathFactory objectPathFactory, IObjectBaseFactory objectBaseFactory)
       {
          _objectPathFactory = objectPathFactory;
          _objectBaseFactory = objectBaseFactory;
       }
 
-      public NormalDistributionFormula CreateNormalDistributionFormulaFor(IDistributedParameter distributedParameter, IParameter meanParameter, IParameter deviationParameter)
+      public NormalDistributionFormula CreateNormalDistributionFormulaFor(IDistributedParameter distributedParameter, IParameter meanParameter, IParameter deviationParameter) =>
+         createDistribution<NormalDistributionFormula>(meanParameter.Dimension, configureNormalDistribution(meanParameter.Name, deviationParameter.Name));
+
+      private Action<DistributionFormula, IDimension> configureNormalDistribution(string meanParameterName = null, string deviationParameterName = null) => (distributionFormula, dimension) =>
```

Comment:
I think `null `as default value for mean and deviation parameter names does not make sense.
Both parameters must be either mandatory or the default value should be the default names of those parameters 
(which are "Mean" and "Deviation" I think - but needs to be checked)